node/src/components/small_network/limiter.rs

```diff
@@ -456,6 +469,13 @@ mod tests {
         let diff = end - start;
         assert!(diff >= Duration::from_secs(5));
         assert!(diff <= Duration::from_secs(6));
+
+        // Ensure metrics recorded the correct number of seconds.
+        println!("wait_metric is {}", wait_metric.get());
```

Comment:
Should remove this `println!` and the one on line 516.

Reply:
I put it in because usually something like `assert_eq!` would print a helpful message, but since I have to check for a range, it just says "failed" without giving the concrete value.

However I just realized that this could be 

```rust
assert!(wait_metric.get() <= 6.0, "metric too large: {}", wait_metric.get());
```

instead. I'll update it.